courses/free-courses-pt_BR.md

```diff
@@ -395,6 +396,13 @@
 * [Fundamentos de Sistemas Embarcados](https://www.youtube.com/playlist?list=PLqvo6YdcIqXXGY1dLbf_xA-JLMBumTyzG) - Renato Sampaio
 
 
+### Smalltalk
+
+* [Conhecendo o SmallTalk](https://www.researchgate.net/publication/262882317_Conhecendo_o_Smalltalk_-_Todos_os_Detalhes_da_Melhor_Linguagem_de_Programacao_Orientada_a_Objetos) - Daniel Duarte Abdala e
+Aldo von Wangenheim (PDF)
```

Comment:
need to have this all on one line

Reply:
and separate authors with a comma

Reply:
@eshellman, I made the changes as requested.